docs/source/deploy.rst

```diff
@@ -0,0 +1,151 @@
+======
+Deploy
```

Comment:
To follow the titles from the other sections ("Installing Voila", "Using Voila") we could maybe rename it to something like:

```suggestion
Deploying Voila
```

Reply:
done